src/Ilios/CoreBundle/Entity/Repository/AuditLogRepository.php

```diff
@@ -19,10 +19,10 @@ class AuditLogRepository extends EntityRepository
     public function findInRange(\DateTime $from, \DateTime $to)
     {
         $qb = $this->_em->createQueryBuilder();
-        $qb->add('select', 'a')
+        $qb->select('a as log', 'u.id as userId')
```

Comment:
No because it isn't SQL against MySQL it is DQL against the doctrine User entity.  That class maps the DB column user_id to the property id.